src/PrestaShopBundle/Form/Admin/Sell/Product/Pricing/RetailPriceType.php

```diff
@@ -127,28 +152,50 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'modify_all_shops' => true,
             ])
             ->add('tax_rules_group_id', ChoiceType::class, [
-                'choices' => $this->taxRuleGroupChoices,
+                'choices' => $this->taxRuleGroupChoicesProvider->getChoices(),
                 'required' => false,
                 // placeholder false is important to avoid empty option in select input despite required being false
                 'placeholder' => false,
-                'choice_attr' => $this->taxRuleGroupChoicesAttributes,
+                'choice_attr' => $this->taxRuleGroupChoicesProvider->getChoicesAttributes(),
                 'attr' => [
                     'data-toggle' => 'select2',
                     'data-minimumResultsForSearch' => '7',
-                    'data-tax-enabled' => $this->taxEnabled,
+                    'data-tax-enabled' => $this->isTaxEnabled,
                 ],
                 'row_attr' => [
                     'class' => 'retail-price-tax-rules-group-id',
                 ],
                 'label' => $this->trans('Tax rule', 'Admin.Catalog.Feature'),
-                'help' => !$this->taxEnabled ? $this->trans('Tax feature is disabled, it will not affect price tax included.', 'Admin.Catalog.Feature') : '',
+                'help' => !$this->isTaxEnabled ? $this->trans('Tax feature is disabled, it will not affect price tax included.', 'Admin.Catalog.Feature') : '',
```

Comment:
It's weird. If the tax is disabled, it seems clear that the price (including tax) will not be affected... but neither will the price excluding tax. If there is no tax? I find this message confusing and don't understand what the message is.

Reply:
I don't know it's been here for a while, it was not the purpose of the PR, only the variable name was changed If you think it's useless we can clear it But maybe in another I don't want to block this one because of unrelated wording

If I have other bug to fix in the PR I will do it, if not we can clear it when we'll perform the wording review on the whole page